index.js

```diff
@@ -2,24 +2,54 @@
 
 // Adapted from https://github.com/markdown-it/markdown-it/blob/fbc6b0fed563ba7c00557ab638fd19752f8e759d/docs/architecture.md
 
-function markdownitLinkAttributes (md, config) {
-  config = config || {}
+var optionKeys = ['ignoreRelative', 'pattern']
```

Comment:
Let's put these in screaming snake case to indicate they are constants:

```js
var OPTION_KEYS = ['ignoreRelative', 'pattern']
var ABSOLUTE_URL_PATTERN = /^([a-z]+:)?\/\//
 ```

Reply:
I see :)